tests/Fixtures/Util/ThrowingProcessExecutorMock.php

```
@@ -2,29 +2,15 @@
 use Composer\Util\ProcessExecutor;
+use RuntimeException;
 
-/**
- * Mock of ProcessExecutor that always throws.
- *
- * @copyright Copyright (C) 2026 Terabytesoftw.
- * @license https://opensource.org/license/mit/ MIT License
- */
 final class ThrowingProcessExecutorMock extends ProcessExecutor
 {
     public function execute($command, &$output = null, string|null $cwd = null): int
```

PHPMD (2.15.0)

Avoid unused parameters such as '$command'. (undefined)

(UnusedFormalParameter)